import/index.template.html

```diff
@@ -636,6 +643,28 @@
             // In the case of import, this public key is not used because the client encrypts messages
             // to the server's public key.
             TKHQ.sendMessageUp("PUBLIC_KEY_READY", "");
+
+            const plaintextTextarea = document.getElementById("plaintext");
+            if (!plaintextTextarea) return;
+
+            plaintextTextarea.addEventListener("paste", async () => { 
+              // Let the paste happen first
+              setTimeout(async () => {
```

Comment:
can we test with/without this set timeout? I'd imagine this function is called as a callback to the paste, so will always happen strictly after the paste is finished. but could be wrong here